### DISCUSSION TITLE
Proposal-Review

### DISCUSSION THREAD

**Discussion by mumin-khan** [camicroscope/GSOC#37] [Q&A] [answered]
@nanli-emory I have drafted my proposal for Collection and Study Management. Where should I share it for the review?

**Comment by shaiktahseen** [comment]
@MuminAhmadKhan bro can you help me how to submit proposal for review i see last date is 4th april 2023.

If you can share you linkedln profild will discuss there.

**Reply by mumin-khan** [reply]
@shaiktahseen I have asked the mentors. Lets wait for their response.

**Comment by shaiktahseen** [comment]


**Reply by mumin-khan** [reply]
https://github.com/camicroscope/GSOC/discussions/26#discussioncomment-5482144
This might help

**Comment by shaiktahseen** [comment]
Ok

**Comment by nanli-emory** [comment, marked as answer]
Submit your proposal early during the application period so that the organization mentors can review it and ask you questions or request more detail on aspects of your proposal before the final deadline. **You can edit the proposal as many times as you wish before the application deadline.** You may even want to label the proposal as a draft so it is clear you are looking for feedback before submitting the final proposal.

Remember, thousands of potential GSoC contributors are submitting proposals so it can take organizations a few days or even a week+ to get back to you if they have questions. So the earlier you submit a well-written draft proposal, the more time they have to give you feedback on it so you can make it stronger and understand more of what they are looking for.

Follow the instructions from the organizations on the content and format of your proposal.

**Reply by shaiktahseen** [reply]
@nanli-emory hey i dont know how to submit proposal i see last date to submit proposal 4th april 2023.
I'm a beginner can you help let me know how to submit proposal what to write, what not to write, how to structure etc... that would be really helpful to me.

**Comment by shaiktahseen** [comment]
@MuminAhmadKhan bro I'm a beginner can you help let me know how to submit proposal what to write, what not to write, how to structure etc... that would be really helpful to me.
 i dont know how to submit proposal i see last date to submit proposal 4th april 2023.

**Comment by shaiktahseen** [comment]
Bro I'm a beginner can you help let me know how to submit proposal what to
write, what not to write, how to structure etc... that would be really
helpful to me.
i dont know how to submit proposal i see last date to submit proposal 4th
april 2023.

**Reply by gaurab-khanal** [reply]
I also need guidance in this.

**Reply by shaiktahseen** [reply]
@gaurab-khanal ok

**Reply by mumin-khan** [reply]
https://www.youtube.com/watch?v=LmVolos4OMA
This will be useful

**Reply by shaiktahseen** [reply]
> I also need guidance in this.
> 

What response you got bro,

Have you submitted your proposal to organisations?
